metaflow/plugins/kfp/kfp.py

```diff
@@ -61,13 +74,16 @@ def __init__(
         self.username = username
         self.base_image = base_image
         self.s3_code_package = s3_code_package
+        self.max_parallelism = max_parallelism
+        self.workflow_timeout = workflow_timeout if workflow_timeout else 0
```

Comment:
This means we need to set a timeout, or the run will immediately exit. Can we set a better default timeout in case the AS forgets to set it? Perhaps 10 minutes.